website/docs/reference/appsmith-framework/query-object.mdx

```diff
@@ -34,14 +41,13 @@ This function returns a JavaScript **promise**, which can be used to handle asyn
         .catch(() => {...}) // run if the query encounters any errors
 }}
 ```
+It can't be used in [synchronous fields](/core-concepts/writing-code/workflows#display-data-from-async-js-function).
 
-To learn more about chaining actions to create complex workflows, see [complex workflows](/core-concepts/writing-code/workflows#complex-workflows).
-
-### Pass parameters to run() {#pass-parameters-to-run}
+</dd>
 
-In some cases, like [running a query inside a loop](/connect-data/how-to-guides/how-to-pass-params-to-an-api), parameters may need to be passed to the query with values contextual to the execution.
+#### Pass parameters to run()
```

Comment:
this was supposed to be removed and linked to the how to the guide if the example is the same

Reply:
The example is not same. That's why I kept it. Forgot to mention in the message.